Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
I guess there's no harm in it. If someone's going to choose to build with the static libs enabled, they should expect the binaries to be linked to those static libs. Seems a fair and reasonable assumption. I'm happy to withdraw my light objection if the maintainers are happy. This needs rebasing against the master though.